### PR TITLE
xslt changes

### PR DESCRIPTION
**Pull request by 4akk1**
Hi Peter, as discussed in the ros chat I have created a fork with the relevant changes to xslt.

### CODE REVIEW

File: xslt/styles.xslt [hunk 1]
```diff
@@ -143,7 +144,7 @@
         <xsl:attribute name="font-size">13pt</xsl:attribute></xsl:attribute-set>
     <xsl:attribute-set name="h3" use-attribute-sets="bold h-title">
         <xsl:attribute name="font-size">12pt</xsl:attribute></xsl:attribute-set>
-    <xsl:attribute-set name="h4" use-attribute-sets="bold h-title"></xsl:attribute-set><xsl:attribute-set name="h5" use-attribute-sets="italic h-title"></xsl:attribute-set>
+    <xsl:attribute-set name="h4" use-attribute-sets="bold h-title"></xsl:attribute-set>
```
Comment (PeterMosmans) [inline xslt/styles.xslt:147]: Would you mind making sure that this PR only contains changes relevant for this added functionality ? Atomic PR requests per feature / fix / ... make it easier to review, git-blame, ... . Thanks!

Comment (4akk1) [reply xslt/styles.xslt:147]: Yeah of course, should I have two PR's one for the labels and one for the "Accept" status essentially?

Comment (PeterMosmans) [reply xslt/styles.xslt:147]: Yes please, as atomic as possible, one branch / PR per feature / .. .

Comment (PeterMosmans) [reply xslt/styles.xslt:147]: Thanks a bunch in advance!

Comment (4akk1) [reply xslt/styles.xslt:147]: Of course, I made a new PR which has only the labels/multiple table creation (this one is kind of a 2 in one hope it's ok)

**Pull request closed without merging**